bootstrap/service.go

```diff
@@ -26,18 +26,21 @@ var (
 	// when accessing a protected resource.
 	ErrUnauthorizedAccess = errors.New("missing or invalid credentials provided")
 
+	// ErrAuthorization indicates a failure occurred while authorizing the entity.
+	ErrAuthorization = errors.New("failed to perform authorization over the entity")
+
```

Comment:
`ErrAuthorization` and `ErrUnauthorizedAccess` sound very similar, as well as their respective error messages. I find it confusing and too generic.